docs/sources/alerting/migrating-alerts/opt-in.md

```diff
@@ -33,9 +32,9 @@ Before v8.2, notification logs and silences were stored on a disk. If you did no
 
 To disable Grafana alerts and roll back to legacy dashboard alerting:
```

Comment:
Is there anything we can say here about the impact of disabling and rolling back? What happens to alerts that were defined with the new alerting?

Reply:
You lose them and grafana panics. There is a note about that.